packages/devtools_app/lib/src/screens/network/har_network_data.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/foundation.dart';
+
+import '../../../devtools_app.dart';
+import 'har_builder.dart';
+// ignore_for_file: avoid_dynamic_calls
+
+class HarNetworkData {
+  HarNetworkData(this.requests);
+
+  factory HarNetworkData.fromJson(Map<String, dynamic> json) {
+    final entries = (json['log']?['entries'] as List<dynamic>? ?? [])
+        .map(
+          (entryJson) =>
+              DartIOHttpRequestData.fromJson(entryJson as Map<String, dynamic>),
```

Comment:
use `Object?` everywhere instead of `dynamic`

Reply:
used Object? instead of dynamic in HarNetworkData